### DISCUSSION TITLE
Add ability to sign in to additional organizations

### DISCUSSION THREAD

**Discussion by devonbleak** [community/community#30474] [Mobile]
On mobile I don't seem to be able to authorize additional organizations without completely signing out and signing back in.

**Comment by taki-on** [comment]
This is true. Authorizing an organization is only possible during the sign-in process. We will discuss if there is a way to authorize additional organizations without signing out. Thank you for your feedback!

**Reply by 40041588** [reply]
```
W6A4ZESMJQDPKBPF765TDKIUJ7EXUTDC
```

**Comment by JackFreelander** [comment]
It's somewhat misleading, because you can "pull down" and refresh the organizations in the mobile app, but the app doesn't even tell you that there are additional ones you need to authorize.  If it's too heavy a lift to authorize additional GitHub organizations without signing out, perhaps at least a breadcrumb/hint for users that this is required?
